sphinx_issues.py

```diff
@@ -344,41 +344,39 @@ def setup(app):
         rebuild="html",
         types="[str]",
```

Comment:
issue: doesn't this need to be changed, too?

```suggestion
        types=[str],
```